README.md

```diff
@@ -1043,7 +1043,7 @@ INGESTION           0.39        0.10        0.12        0.13        0.15
 
 Before running this mode, you need to use the correctness write mode to write data to the database.
 
-In order to verify the correctness of the data set writing, you can use this mode to query the data set written to the database. Currently this mode only supports IoTDB v0.12 and v0.13
+In order to verify the correctness of the data set writing, you can use this mode to query the data set written to the database. Currently this mode only supports IoTDB v0.12 , v0.13 and influxdb v1.x
```

Comment:
Please update README-cn.md at the same time.

Reply:
Done~